Sources/DependenciesMacrosPlugin/DependencyEndpointMacro.swift

```diff
@@ -162,14 +162,14 @@ public enum DependencyEndpointMacro: AccessorMacro, PeerMacro {
       }
       let appliedParameters =
         parameters
+        .enumerated()
         .map {
-          guard let typed = $0.type.as(AttributedTypeSyntax.self),
-            typed.specifier?.tokenKind == .keyword(.inout)
-          else { return false }
-          return true
+          $1.isInout
+          ? "&p\($0)"
+          : $1.isAutoclosure
+          ? "p\($0)()"
```

Comment:
Only `()` is OK because `@autoclosure` can be used only when there is no parameters.
Also `inout` expression cannot be used on `@autoclosure` so we don't need to consider the situation when `$1.isInout && $1.isAutoclosure`.